tools/gulp/tasks/docs.ts

```diff
@@ -0,0 +1,16 @@
+import gulp = require('gulp');
+const markdown = require('gulp-markdown');
+const transform = require('gulp-transform');
+
+
+const EXAMPLE_PATTERN = /<!--\W*example\(([^)]+)\)\W*-->/g;
```

Comment:
Can you add a comment explaining what we're replacing and why we're replacing it?